<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_ingame</class>
 <widget class="QWidget" name="widget_ingame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>659</x>
     <y>-10</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/frame/resources/frame/time.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="description">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>20</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(85, 85, 85)</string>
   </property>
   <property name="text">
    <string>关卡进度</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="progress">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>40</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(85, 85, 85)</string>
   </property>
   <property name="text">
    <string>0 / 0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressbar">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>70</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {
    border: 3px solid rgb(255, 255, 255);

    background-color: rgba(255,255,255,0.5);
    text-align: center;
    color: white;
    font-weight: bold;
}

QProgressBar::chunk {
    background-color:rgb(76, 175, 80)
}
</string>
   </property>
   <property name="value">
    <number>20</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="IconButton" name="pause">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background: transparent;       /* 背景透明 */
        border: none;                  /* 无边框 */
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/button/resources/button/begin.png</normaloff>:/button/resources/button/begin.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="IconButton" name="speed">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background: transparent;       /* 背景透明 */
        border: none;                  /* 无边框 */
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/button/resources/button/speed.png</normaloff>:/button/resources/button/speed.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
        background-color: rgba(150, 150,150,128); /* 浅灰 */
        border-radius: 4px; /* 圆角（可选） */
    }
    /* 鼠标悬停状态：加深为深灰色 */
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="stability_progressbar">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {
    border: 3px solid rgba(220, 220, 220, 1) ;
    background-color: rgba(255,255,255,0.1);
    text-align: center;
    color: white;
    font-weight: bold;
}

QProgressBar::chunk {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 rgba(116, 235, 213, 0.8),
        stop:1 rgba(172, 182, 229, 0.8));
    border: 1px solid rgba(255,255,255,0.5);
}</string>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="stability_value">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>100</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
   </property>
  </widget>
  <widget class="IconButton" name="menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background: transparent;       /* 背景透明 */
        border: none;                  /* 无边框 */
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/button/resources/button/menu.png</normaloff>:/button/resources/button/menu.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QLCDNumber" name="resource_value">
   <property name="geometry">
    <rect>
     <x>53</x>
     <y>50</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color:white;  
       

border-radius: 6px;       
padding: 5px;            </string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::SegmentStyle::Flat</enum>
   </property>
   <property name="intValue" stdset="0">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="resource">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/resource/resources/resource/Inspiration.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="stability">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/resource/resources/resource/Dream_Stability.png</pixmap>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>description</zorder>
  <zorder>progressbar</zorder>
  <zorder>pause</zorder>
  <zorder>speed</zorder>
  <zorder>label_2</zorder>
  <zorder>stability_progressbar</zorder>
  <zorder>stability_value</zorder>
  <zorder>progress</zorder>
  <zorder>menu</zorder>
  <zorder>resource_value</zorder>
  <zorder>resource</zorder>
  <zorder>stability</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IconButton</class>
   <extends>QPushButton</extends>
   <header>IconButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
